smoke-test/device_manager_smoke_test.js

```diff
@@ -16,7 +16,7 @@
 
 describe('DeviceManagerSmokeTest', () => {
   if (!process.env.GCLOUD_PROJECT) {
-    throw new Error('Usage: GCLOUD_PROJECT=<project_id> node #{$0}');
+    throw new Error("Usage: GCLOUD_PROJECT=<project_id> node #{$0}");
```

Comment:
@alexander-fenster weird, it looks like gapic is using double quotes here.  This is weird for a few reasons:
1. The generator should probably use single quotes :)
2. I'm surprised `npm run prettier` didn't fix this

@alexander-fenster can you check out what's up here?

Reply:
```
➜  nodejs-iot git:(autosynth) ✗ npm run prettier

> @google-cloud/iot@0.1.1 prettier /Users/jonathanlui/nodejs-iot
> prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js
```

smoke-test/ directory isn't linted.

Reply:
Raises a better question - why do we have a smoke-test directory at all? Couldn't we just lump these into the system tests?